util/set_android_debuggable.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+INPUT="$1"
+
+if [ "$BUILDTYPE" == "Debug" ] ; then
+	sed -i.bak 's/android:debuggable="false"/android:debuggable="true"/g' "${INPUT}"
```

Comment:
I'm not 100% sure that this sed command will work with GNU sed (i.e on Linux). Unfortunately, the Vulcan build won't test it as it uses $BUILDTYPE==Fast.

Reply:
I _think_ the sed command is ok but it didn't seem to do the actions in order so I'm combing the copy and set debuggable actions into the one script

Reply:
@montegoulding I take it this was actioned? (Its not clear what change you suggested)

Reply:
@runrevmark I think so but I'll double-check.

Reply:
aha... still have it checked out on my linux VM and I managed to clobber my tweak when I rebased the branch on mac so will sort that in a sec